--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@
 
 def find_boost():
     """Find the name of the boost-python library. Returns None if none is found."""
-    boostlibnames = ['boost_python-py%s%s' % (sys.version_info[0], sys.version_info[1])]
+    boostlibnames = ['boost_python%s%s' % (sys.version_info[0], sys.version_info[1])]
     boostlibnames += ['boost_python']
     if sys.version_info[0] == 2:
         boostlibnames += ["boost_python-mt"]
